data_specification/constants.py

```diff
@@ -29,23 +29,23 @@
 
 # DSG Arrays and tables sizes:
 #: Maximum number of registers in DSG virtual machine.
-MAX_REGISTERS = 16
+MAX_REGISTERS = 32
 #: Maximum number of memory regions in DSG virtual machine.
-MAX_MEM_REGIONS = 16
+MAX_MEM_REGIONS = 32
```

Comment:
Ah I now see what you mean about the "other" values... I think it is actually only this one that needs to change (the others are almost unused in any code we have, but probably best not to change them anyway).